src/test/statistics/chunk_statistics/range_filter_test.cpp

```diff
@@ -31,17 +31,17 @@ class RangeFilterTest : public ::testing::Test {
     _after_range = _max_value + 1;   // value larger than the maximum
   }
 
-  std::shared_ptr<RangeFilter<T>> test_varying_range_filter_size(size_t gap_count, pmr_vector<T> value) {
+  std::shared_ptr<RangeFilter<T>> test_varying_range_filter_size(size_t gap_count, pmr_vector<T> values) {
     // RangeFilter constructor takes range count, not gap count
-    auto filter = RangeFilter<T>::build_filter(_values, gap_count + 1);
+    auto filter = RangeFilter<T>::build_filter(values, static_cast<uint32_t>(gap_count + 1));
 
-    for (const auto& value : _values) {
+    for (const auto& value : values) {
```

Comment:
@timzimmermann @Bouncner - I found this while working on the PR, but it does not actually seem to have anything to do with Weverything.

Previously `values` was ignored and `test_varying_range_filter_size` erroneously used `_values`. I changed that, but now the tests fail. So far, I am not sure if the test or the actual implementation is broken. Could you please have a look?

Reply:
So I checked and I am convinced there are some issues in the tests. I started fixing some of them (adf7944), but there are a few things I want to talk to @Bouncner about to validate my reasoning.
We will update this thread once that is done.

Reply:
Talked to @Bouncner, he will address the remaining stuff one of these days.
We agree that the test does not make any sense in its current form. He will try to think of a better way to test the general idea of large value domains.